toolset/benchmark/benchmarker.py

```diff
@@ -396,20 +396,31 @@ def __run_test(self, test):
                                                     "ERROR: Problem starting")
                     return sys.exit(1)
 
-                logging.info("Sleeping %s seconds to ensure framework is ready"
-                             % self.config.sleep)
-                time.sleep(self.config.sleep)
+                slept = 0
+                max_sleep = 60
+                while not test.is_running() and slept < max_sleep:
+                    time.sleep(1)
+                    slept += 1
+
+                if not docker_helper.successfully_running_containers(
```

Comment:
If this `if not docker_helper.successfully_running_containers` block was moved into the `while not test.is_running()` block, would that give us early failure when a container dies early?

Reply:
Probably; I'll do that.